internal/generate/crd/crd_test.go

```diff
@@ -283,6 +290,9 @@ spec:
 const crdCustomExpV1beta1 = `apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
+  annotations:
+    operators.operatorframework.io/builder: operator-sdk-unknown
+    operators.operatorframework.io/project_layout: unknown
   name: memcacheds.cache.example.com
```

Comment:
Unfortunately, I have no opportunity to check the doc to raise it up and I still think that might would be better we follow up the current layout. However, since it was approved in the enhancement doc let's move forward with. You are right 👍